Gaijinizer patch/Map164.txt

```diff
@@ -21,146 +22,160 @@ because of yourself. You made it a "monster".#MTLed
 もっとも、暴走した魔力で世界を滅亡させる
 『混沌』を呼ぶメシアだけどね。
 > CONTEXT: Map164/events/1/pages/1/13/Dialogue/hyakki
-If you leave it,\\C[2] You become "Savior (Messiah)".
-However, the world is destroyed by the runaway
-magic power. It is a messiah that calls "chaos".#MTLed
+If she leaves you be, \\C[2]you'll turn into this
+"Messiah" and destroy the world with your runaway
+power. You'll only be the "Messiah" of chaos,
+though.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 それを防ぐには、『天使の魔力』を取り除くしかない……
 
 だけど、それにはアンタを殺さなきゃならない。
 > CONTEXT: Map164/events/1/pages/1/18/Dialogue/hyakki
-The only way to prevent that is to remove
-"Angel's magic"...But I have to kill you.#MTLed
+The only way to prevent that is to extract the
+angelic magic out of you, but she'll have to kill
+you to do that.#Ron
 > END STRING
 
 > BEGIN STRING
 いや……
 
 \\C[2]今のアンタには、もう一つ選択肢があるかもしれないネ
 > CONTEXT: Map164/events/1/pages/1/25/Dialogue/hyakki
-Disagreeable……. \\C[2]There may be another option for
-you now#MTLed
+Well actually...
+
+\\C[2]The way you are now, there might be another
+option.#Ron
 > END STRING
 
 > BEGIN STRING
 もう一つの……『選択肢』？
 > CONTEXT: Map164/events/1/pages/1/29/Dialogue/kazuya
-Another..."choice"?#MTLed
+Another... "option"?#Ron
 > END STRING
 
 > BEGIN STRING
 アンタは、\\C[2]出会った悪魔を一体も殺してない。\\C[0]
 
 つまり、天使の魔力も暴走しようがないのサ。
 > CONTEXT: Map164/events/1/pages/1/31/Dialogue/hyakki
-You are\\C[2] I have not killed one of the demons I
-met.\\C[0]. In other words, the angel's magic can not
-run away.#MTLed
+\\C[2]You haven't killed a single demon you've met so
+far\\C[0]. In other words, the angelic magic can't
+run riot.#Ron
 > END STRING
```

Comment:
"riot" -> "amok" (Riot still works here, but I think this is a better word choice, feel free to add your thoughts on this).